docs/UserGuide.md

```
@@ -91,12 +91,56 @@ title: User Guide
-Format: `help`
+Room search:
+* Must match exact format `#BLOCK-ROOM-LETTER` (e.g., `#14-203-D`)
 
+Warnings:
+No input provided → `Invalid command format!`
+
+Tips:
+* Use multiple keywords to quickly find groups.
+* After `find`, you can use `edit` or `delete` using the indices shown in the result list.
+
+---
```

markdownlint-cli2 (0.22.0)

[warning] Horizontal rule style
Expected: --------------------------------------------------------------------------------------------------------------------; Actual: ---

(MD035, hr-style)